### PR TITLE
Graphivz dot output

### PR DESCRIPTION
**Pull request by omega16**
Now you can choose between write a png, write a dot file or get a pydot graph from your tree

### CODE REVIEW

**Comment by omega16** [comment]
This solve issue #498

**Comment by MegaIng** [comment]
You can just refactor all tree methods to the tree class, in the part that is not included in the standalone parser.

This will make it clearer: These functions aren't useful without the Tree. Just make sure to keep the original method for backwards compatibility.

Unless @erezsh disagree and says that these functions are for debugging, and should therefore be kept separated from the `Tree` class.

**Comment by omega16** [comment]
I think more or less the same, only problem is backwards compatibility, so, marking pytdot_tree_to_png as deprecated, leave that function (for now).

new function names could be one of :


~~~python
write_dot
write_png
get_pydot_graph
get_graphviz_graph

save_dot
save_png

to_dot
to_png
to_graphviz_graph

tree_to_dot
tree_to_png
tree_to_graphviz

~~~

File: examples/fruitflies.py [hunk 1]
```diff
@@ -33,9 +33,13 @@
 def make_png(filename):
     tree.pydot__tree_to_png( parser.parse(sentence), filename)
 
+def make_dot(filename):
+    tree.pydot__tree_to_dot( parser.parse(sentence), filename)
+
 if __name__ == '__main__':
     print(parser.parse(sentence).pretty())
     # make_png(sys.argv[1])
+    # make_dot(sys.argv[1])
```
Comment (erezsh) [inline examples/fruitflies.py:42]: Why two comments?

**Review by erezsh — COMMENTED** [review]
I'm ambivalent about it. If you want to move it to Tree, that's okay by me.

File: examples/fruitflies.py (hunk 1 above)
Comment (omega16) [reply examples/fruitflies.py:42]: It was to expose the use of dot output, but now it seems that defining `make_do` is enougth.

**Comment by erezsh** [comment]
Thanks for the PR!

**Pull request merged**